src/util/html.js

```diff
@@ -0,0 +1,97 @@
+import { camelToDash } from './AttributeParser';
+
+class Part {
```

Comment:
@lukas-schardt What is the difference between Part and TemplatePart?

Reply:
@eddyloewen `Part` is the base class, where `TemplatePart` inherits from with only real basic encoding / toString functionality. But they could be merged

Reply:
Oh I see it now being used for Attributes. I thought it wasn't used but only for subclassing. I think it's fine.